stage #2.5/module #6 [Theory] Architecture advanced - MVC, 3-layered, gradle modules/#1. Layered Architecture/content.md

```diff
@@ -0,0 +1,46 @@
+<H1 style="color: green">Layered Architecture</h1>
+
+***
```

Comment:
this *** create strange grey line. Please remove it

Reply:
But it strange. Can't see this locally.